rust-runtime/aws-smithy-http-server-python/src/server.rs

```diff
@@ -73,17 +77,15 @@ pub trait PyApp: Clone + pyo3::IntoPy<PyObject> {
                     .getattr(py, "pid")
                     .map(|pid| pid.extract(py).unwrap_or(-1))
                     .unwrap_or(-1);
-                tracing::debug!("Terminating worker {idx}, PID: {pid}");
+                println!("Terminating worker {idx}, PID: {pid}");
```

Comment:
These tracing statements have been moved to `println!` since at this stage we do not have the tracing subscriber initialized.